charts/scalardb-graphql/Chart.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: v2
+name: scalardb-graphql
+description: Scalar DB GraphQL server
+type: application
+version: 1.0.0
+appVersion: 3.5.0
+icon: https://scalar-labs.com/wp-content/themes/scalar/assets/img/logo_scalar.svg
+keywords:
+- scalardb
+- graphql
+home: https://scalar-labs.com/
+sources:
+  - https://github.com/scalar-labs/scalardb
```

```suggestion
  - https://github.com/scalar-labs/scalardb-graphql
```
I think the sources URL (repository name) is `scalardb-graphql`.